hyperlane-mailbox/src/main.sw

```diff
@@ -64,6 +70,47 @@ impl Mailbox for Contract {
         message_id
     }
 
+    #[storage(write)]
+    fn set_default_ism(module: ContractId) {
```

Comment:
Could you add a TODO here to make this only callable by the owner? Just wanna avoid it slipping between the cracks